src/reactphp/src/Runner.php

```diff
@@ -2,33 +2,23 @@
 
 namespace Runtime\React;
 
-use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
 use Symfony\Component\Runtime\RunnerInterface;
 
 class Runner implements RunnerInterface
 {
-    private $application;
-    private $port;
+    private RequestHandlerInterface $application;
+    private ServerFactory $serverFactory;
 
-    public function __construct($application, $port)
+    public function __construct(ServerFactory $serverFactory, RequestHandlerInterface $application)
```

Comment:
This is a BC Break which we need to discuss. And would require a new `0.x` version to not break things.